### DISCUSSION TITLE
FIO will support PI with Storage tag？

### DISCUSSION THREAD

**Discussion by ll123456** [axboe/fio#1745] [Q&A]
[root@localhost scripts]# sudo fio -filename=/dev/ng0n1 -iodepth=128 -rw=write -bs=8208 -numjobs=1 -name=test -direct=0 -ioengine=io_uring_cmd -size=100%  -pi_act=0  -pi_chk=GUARD,APPTAG,REFTAG -group_reporting -buffer_compress_chunk=4k  -buffer_compress_percentage=1 --runtime=10
test: (g=0): rw=write, bs=(R) 8208B-8208B, (W) 8208B-8208B, (T) 8208B-8208B, ioengine=io_uring_cmd, iodepth=128
fio-3.36
Starting 1 process
/dev/ng0n1: Storage tag not supported

**Comment by vincentkfu** [comment]
I know of no current plans to add storage tag support.

**Comment by ll123456** [comment]
I got it, Thank you for your reply
